src/React/Widgets/AnnotationStoreEditorWidget/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import style from 'PVWStyle/ReactWidgets/AnnotationStoreEditorWidget.mcss';
+
+import ActionListWidget from '../ActionListWidget';
+import AnnotationEditorWidget from '../AnnotationEditorWidget';
+import ToggleIconButtonWidget from '..//ToggleIconButtonWidget';
+
+function button(label, action) {
+  return <div key={label} className={style.button} onClick={action}>{label}</div>;
+}
+
+export default function annotationStoreEditorWidget(props) {
+  if (!props.annotation || !props.annotations) {
+    return null;
+  }
+
+  const listAnnotation = Object.keys(props.annotations).map((name, index) =>
+    ({
+      name,
+      action: `${index}`,
+      data: `${name}`,
+      active: (props.annotation === props.annotations[name]),
+    }));
+
+  const onActivateAnnotation = (name, action, data) => {
+    props.onChange('select', name, props.annotations[name]);
+  };
+
+  const storeAction = name => (() => props.onChange(name, props.title, props.annotation));
```

Comment:
// Create a storeAction callback builder

Would that be enough? If not, you should come up with something that make more sense to you.